templates/ansible/hieradata.j2

```diff
@@ -75,10 +69,13 @@ aem_curator::install_dispatcher::apache_module_tarball: dispatcher-apache2.4-lin
 aem_curator::install_dispatcher::apache_module_filename: dispatcher-apache2.4-{{ aem.dispatcher.version }}.so
 aem_curator::install_dispatcher::data_volume_device: "{{ aws[os_type].devicemap_data_b }}"
 
-aem_curator::install_java::jdk_base_url: "{{ aem.jdk.base_url }}"
-aem_curator::install_java::jdk_filename: "{{ aem.jdk.filename }}"
-aem_curator::install_java::jdk_version: "{{ aem.jdk.version }}"
-aem_curator::install_java::jdk_version_update: "{{ aem.jdk.version_update }}"
+aem_curator::install_java::jdk_base_url: "{{ java.base_url }}"
+aem_curator::install_java::jdk_filename: "{{ java.filename }}"
+aem_curator::install_java::jdk_version: "{{ java.version }}"
```

Comment:
Please update the packer-aem default configuration and add these config parameters. If you add them please refer them to the existing parameters e,g, `{{ aem.jdk.base_url }}` to keep the backward compatibility Thanks

Reply:
Done